integrations/operator/controllers/resources/secretlookup/secretlookup.go

```diff
@@ -35,9 +35,9 @@ const (
 	secretScheme = "secret"
 	secretPrefix = secretScheme + "://"
 
-	// AllowInclusionAnnotation is the annotation a secret must wear for the operator to allow looking up its content
+	// AllowLookupAnnotation is the annotation a secret must wear for the operator to allow looking up its content
 	// when reconciling a resource. Its value is either a comma-separated list of allowed resources, or a '*'.
-	AllowInclusionAnnotation = "resources.teleport.dev/allow-inclusion-from-cr"
```

Comment:
Was this not used for anything? Just wondering about backwards compatibility since the annotation name changed here.

Reply:
No, I merged the `secretlookup` lib 2 days ago and nothing uses it yet.